internal/rbd/rbd_util.go

```diff
@@ -1932,3 +1944,26 @@ func (rv *rbdVolume) setAllMetadata(parameters map[string]string) error {
 
 	return nil
 }
+
+// unsetAllMetadata unset all the metadata from arg keys on RBD image.
+func (rv *rbdVolume) unsetAllMetadata(keys []string) error {
+	list, err := rv.ListMetadata()
+	if err != nil {
+		return fmt.Errorf("failed to list metadata image %s/%s: %w", rv.Pool, rv.RbdImageName, err)
+	}
+
+	for k, v := range list {
+		for _, key := range keys {
+			if k != key {
+				continue
+			}
+			err = rv.RemoveMetadata(key)
+			if err != nil && !strings.Contains(err.Error(), "No such file or directory") {
+				return fmt.Errorf("failed to unset metadata key %q, value %q on image %s/%s: %w",
+					k, v, rv.Pool, rv.RbdImageName, err)
```

Comment:
just pass `rv` in `%q` as that will return a string with pool/namespace/imagename